src/styles/bsidestlv.scss

```diff
@@ -140,5 +140,35 @@
 a.btn.primary {
   background-color: grey;
   font-size: 3.5rem;
-  border: solid;
+  border: none;
 }
+
+.divTable{
```

Comment:
If these are only used in CTF page move it to the CTF scss

Reply:
not sure we will only use them there, it depends if we need to build a normal looking div table somewhere else

Reply:
we can do that, but we need to add it to the build process (and lookup in the theme)